coral_credits/api/views.py

```diff
@@ -75,6 +75,88 @@ def retrieve(self, request, pk=None):
+                "auth_url": "https://api.example.com:5000/v3",
+                "region_name": "RegionOne"
+            },
+            "current_lease": {
+                # TODO(assumptionsandg): "lease_id": "bd9f7fe2-39f0-4c05-bf9c-ff5e76cccef5"
+                "start_date": "2020-05-13T00:00:00.012345+02:00",
+                "end_time": "2020-05-14T23:59:00.012345+02:00",
+                "reservations": [
+                {
+                    "resource_type": "physical:host",
+                    "min": 1,
+                    "max": 2,
+                    "hypervisor_properties": "[]",
+                    "resource_properties": "[\"==\", \"$availability_zone\", \"az1\"]",
+                    "allocations": [
+                    {
+                        "id": "1",
+                        "hypervisor_hostname": "32af5a7a-e7a3-4883-a643-828e3f63bf54",
+                        "extra": {
+                        "availability_zone": "az1"
+                        }
+                    }
+                    # TODO(assumptionsandg): "resource_requests" : 
+                    {
+                        "vcpu" / "pcpu" : "8"
+                        "memory" : "4096" # MB ?
+                        "storage" : "30" # GB ?
+                        "storage_type" : "SSD"
+                    }
+                    ]
+                }
+                ]
+            },
+            "lease": {
+                # TODO(assumptionsandg): "lease_id": "e96b5a17-ada0-4034-a5ea-34db024b8e04"
+                "start_date": "2020-05-13T00:00:00.012345+02:00",
+                "end_time": "2020-05-14T23:59:00.012345+02:00",
+                "reservations": [
+                {
+                    "resource_type": "physical:host",
+                    "min": 2,
+                    "max": 3,
+                    "hypervisor_properties": "[]",
+                    "resource_properties": "[\"==\", \"$availability_zone\", \"az1\"]",
+                    "allocations": [
+                    {
+                        "id": "1",
+                        "hypervisor_hostname": "32af5a7a-e7a3-4883-a643-828e3f63bf54",
+                        "extra": {
+                        "availability_zone": "az1"
+                        }
+                    },
+                    {
+                        "id": "2",
+                        "hypervisor_hostname": "af69aabd-8386-4053-a6dd-1a983787bd7f",
+                        "extra": {
+                        "availability_zone": "az1"
+                        }
+                    }
+                    ]
+                    # TODO(assumptionsandg): "resource_requests" : 
+                    {
+                        "vcpu" / "pcpu" : "8"
+                        "memory" : "4096" # MB ?
+                        "storage" : "30" # GB ?
+                        "storage_type" : "SSD"
+                    }
+                }
+                ]
+            }
+            }
+        """
```

Comment:
I think we should be able to have the same request format standardised across all resource providers?

@assumptionsandg I've added some TODOs on what additions we need to the request that blazar sends to the coral-credits API, perhaps @JohnGarbutt has some suggestions on this